companion/lib/main.ts

```diff
@@ -143,12 +143,20 @@ program.command('start', { isDefault: true, hidden: true }).action(() => {
 	}
 
 	// copy an older db if needed
-	if (configDir !== rootConfigDir && !fs.existsSync(path.join(configDir, 'db'))) {
+	if (
+		configDir !== rootConfigDir &&
+		!fs.existsSync(path.join(configDir, 'db')) &&
```

Comment:
does this need to check for 'db' or should it just be 'db.sqlite'?

Reply:
It needs to be `db` still if they leapfrog major updates.